### PR TITLE
Replaces most mentions of GCC with TerraGov

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

Most names that used GCC are now replaced with TerraGov.
Lore about GCC is replaced as well.
"New Earth" is added as a possible home world in character background info, in case you want to make it clear that you are one of the more liberal Terrans.

## Why It's Good For The Game

Part 1 of replacing Bay lore with Tegu lore.

## Did you test it?

I actually did, works fine.

## Changelog

:cl:
tweak: GCC is gone and TerraGov is now in its place! Terran now means quite a different thing, dear lore people.
rscadd: "New Earth" is now added as a possible character homeworld.
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
Rowtree doesn't seem to have any issues with these changes. Consider it approved on the lore side.
image

File: maps/torch/items/clothing/terran-accessory.dm [hunk 1]
```diff
@@ -14,73 +14,69 @@ ranks - TCC
...
 /obj/item/clothing/accessory/terran/rank/navy/flag/o9
 	name = "ranks (O-9 admiral)"
 	desc = "Insignia denoting the rank of Admiral."
 
 /obj/item/clothing/accessory/terran/rank/navy/flag/o10
-	name = "ranks (O-10 admiral of the independent navy)"
-	desc = "Insignia denoting the rank of Admiral of the Independent Navy."
\ No newline at end of file
+	name = "ranks (O-10 master admiral of the humanity)"
+	desc = "Insignia denoting the rank of Master Admiral of the Humanity."
```
Comment (Cupax3) [inline maps/torch/items/clothing/terran-accessory.dm]: While most other stuff is, uh, subject to Rowtree, I can't help but think `Master Admiral of Humanity` would be better.

Comment (EgorDinamit) [reply maps/torch/items/clothing/terran-accessory.dm]: Ah, the famous English grammar? I'll change it.

**Comment by EgorDinamit** [comment]
Done.

File: code/modules/culture_descriptor/faction/factions_human.dm [hunk 2]
```diff
@@ -37,17 +37,14 @@
...
-	in the Gilgamesh system. Externally heavily militant and isolationist, the GCC, internally, is heavily libertarian, with \
-	a strong focus on independent planetary government with the GCC itself only handling defence, foreign relations and some intergalactic trade. \
-	Increasingly anti non-human, the GCC is in a cold war with the Sol Central Government following  the Gaia Conflict, a large \
-	scale conflict between the SCG and GCC that saw a great deal of death and destruction. In Sol space \"Indies\" as they are known are \
-	heavily mistrusted and in some instances, met with downright hostility. Today the GCC continues to be an economic powerhouse following \
-	a period of steady economic recovery, with most of its exports being manufactured goods, primarily armaments."
-	economic_power = 0.9
+/decl/cultural_info/faction/terragov
+	name = FACTION_TERRAGOV
+	description = "The Terran Government, commonly referred to as TerraGov or TG, is a second most powerful human government in the universe. \
```
Comment (Cupax3) [inline code/modules/culture_descriptor/faction/factions_human.dm]: [...] is `the` second most powerful [...]

File: code/modules/culture_descriptor/culture/cultures_human.dm [hunk 3]
```diff
@@ -130,14 +130,16 @@
...
-	resentful about their loss in the Gaia Conflict. An upbringing on Terra emphasises an odd mix of service to the state, liberalism and militarism."
-	language = LANGUAGE_HUMAN_RUSSIAN
-	economic_power = 0.9
+/decl/cultural_info/culture/human/terran
+	name = CULTURE_HUMAN_TERRAN
+	description = "You are from one of the many worlds within Terran government. \
+	Terrans are the embodiment of what it is to be a 'true' human, according to some sources. \
+	The citizens of Terran Government are employed in many various positions, most of which are \
+	closely associated with politics, however. Terrans today are living in decent cities even by \
+	Earth standards, although not quite yet the same. Every human under Terran rule is equal, and, \
```
Comment (Cupax3) [inline code/modules/culture_descriptor/culture/cultures_human.dm]: [...] although not quite the same yet. [...]

File: code/modules/culture_descriptor/faction/factions_human.dm [hunk 4]
```diff
@@ -37,17 +37,14 @@
...
+	As opposed to SolGov, TerraGov's ideals are those of progressing humanity at any cost, especially at the cost of alien civilizations. \
+	Terran Government prohibits any political parties that oppose the Terran Constitution, and as such there is only so much factions within the senate \
+	and each would rephrase 'progress of humanity' as they see fit, ranging from a more peaceful approach of co-existence with non-hostile alien \
```
Comment (Cupax3) [inline code/modules/culture_descriptor/faction/factions_human.dm]: Either:
[...] only so much factions within the senate can do, where each would rephrase [...]
Or:
[...} only so many factions within the senate and each would [...]

File: code/modules/culture_descriptor/faction/factions_human.dm [hunk 5]
```diff
@@ -37,17 +37,14 @@
...
+	civilizations to much more extreme sides of eternal war with everyone who isn't a part of TerraGov. Despite this, senate is considered a stable \
```
Comment (Cupax3) [inline code/modules/culture_descriptor/faction/factions_human.dm]: [...] the senate is considered [...]

File: code/modules/culture_descriptor/location/locations_human.dm [hunk 6]
```diff
@@ -121,17 +121,28 @@
...
-	particular superpower. Terra is one of the many affiliated planets in the GCC and the one that is perhaps the most important. Terra, much like most of the GCC, is isolationist, with a minimal \
-	non-human presence. Much of Terra is heavily industrialised, with a lower standard of living than on equivalent SCG planets. Much of the population works in services, industry or the military. \
-	Still recovering from an economic crash toward the end of the Gaia conflict, much of Terra is in decline, with public services and works being minimal and much of the population burdened by \
-	increasingly high taxes. This state of affairs has seen the immigration of many Terran citizens to the SCG in recent years, hoping to escape the poor conditions they found themselves in. Despite this,\
-	the people of Terra are a proud and unbroken people who are heavily patriotic and tend to bring this with them wherever they go."
+	description = "Terra - the capital world of Terran Government, planet with an atmosphere close to that of Earth and home to many 'true' humans. \
```
Comment (Cupax3) [inline code/modules/culture_descriptor/location/locations_human.dm]: [...] a planet with an [...]

File: code/modules/culture_descriptor/location/locations_human.dm [hunk 7]
```diff
@@ -121,17 +121,28 @@
...
+	Almost 100% of its population is humans, with rare exception for few Skrell ambassadors, which are arguably the only non-human species that are allowed on Terra. \
```
Comment (Cupax3) [inline code/modules/culture_descriptor/location/locations_human.dm]: [...] is human, with the rare exception of a few Skrell ambassadors [...]
[...] the only non-human species that is allowed [...]

File: code/modules/culture_descriptor/location/locations_human.dm [hunk 8]
```diff
@@ -121,17 +121,28 @@
...
+	Terra is also home to the now famous Terran Senate, the main governmental body of Terran Government, and as such, life on Terra usually implies some sort of political \
```
Comment (Cupax3) [inline code/modules/culture_descriptor/location/locations_human.dm]: [...] home of the [...]

File: code/modules/culture_descriptor/location/locations_human.dm [hunk 9]
```diff
@@ -121,17 +121,28 @@
...
-	ruling_body = "Gilgamesh Colonial Confederation"
-	language = LANGUAGE_HUMAN_RUSSIAN
+	distance = "22 light years"
+	economic_power = 1.2
+	ruling_body = "Terran Government"
+	secondary_langs = list(LANGUAGE_SPACER)
+
+/decl/cultural_info/location/human/new_earth
+	name = HOME_SYSTEM_NEW_EARTH
+	description = "New Earth is a second most important planet within TerraGov territories. \
```
Comment (Cupax3) [inline code/modules/culture_descriptor/location/locations_human.dm]: [...] New Earth is the second [...]

File: code/modules/culture_descriptor/location/locations_human.dm [hunk 10]
```diff
@@ -121,17 +121,28 @@
...
+	Unlike Terra, population of New Earth is usually the least xenophobic towards most species and as such is a home to ideas of the most liberal senate factions. \
```
Comment (Cupax3) [inline code/modules/culture_descriptor/location/locations_human.dm]: [...] and as such home to the ideas [...]

**Review by Cupax3 — DISMISSED** [review]
Haha grammar proofread go brrrr

**Review by Cupax3 — APPROVED** [review]
LGTM
